test/fiskaltrust.ifPOS.Tests/v2/ChargeItemSerializationTests.cs

```diff
@@ -0,0 +1,260 @@
+            Assert.AreEqual(original.CostCenter, deserialized.CostCenter);
+            Assert.AreEqual(original.ProductGroup, deserialized.ProductGroup);
+            Assert.AreEqual(original.ProductNumber, deserialized.ProductNumber);
+            Assert.AreEqual(original.ProductBarcode, deserialized.ProductBarcode);
+            Assert.AreEqual(original.Unit, deserialized.Unit);
+            Assert.AreEqual(original.UnitQuantity, deserialized.UnitQuantity);
+            Assert.AreEqual(original.UnitPrice, deserialized.UnitPrice);
+            Assert.AreEqual(original.Currency, deserialized.Currency);
+            Assert.AreEqual(original.DecimalPrecisionMultiplier, deserialized.DecimalPrecisionMultiplier);
+        }
+
+#if NETSTANDARD2_1_TESTS
+        [Test]
+        public void SystemTextJson_SerializeDeserialize_PreservesAllProperties()
+        {
+            // Arrange
+            var original = CreateTestChargeItem();
+            var options = new JsonSerializerOptions 
+            { 
+                WriteIndented = true,
+                Converters = { new System.Text.Json.Serialization.JsonStringEnumConverter() }
+            };
+
+            // Act
+            var json = System.Text.Json.JsonSerializer.Serialize(original, options);
+            var deserialized = System.Text.Json.JsonSerializer.Deserialize<ChargeItem>(json, options);
+
+            // Assert
+            Assert.AreEqual(original.ftChargeItemId, deserialized.ftChargeItemId);
+            Assert.AreEqual(original.Quantity, deserialized.Quantity);
+            Assert.AreEqual(original.Description, deserialized.Description);
+            Assert.AreEqual(original.Amount, deserialized.Amount);
+            Assert.AreEqual(original.VATRate, deserialized.VATRate);
+            Assert.AreEqual(original.ftChargeItemCase, deserialized.ftChargeItemCase);
+            Assert.AreEqual(original.VATAmount, deserialized.VATAmount);
+            Assert.AreEqual(original.Moment, deserialized.Moment);
+            Assert.AreEqual(original.Position, deserialized.Position);
+            Assert.AreEqual(original.AccountNumber, deserialized.AccountNumber);
+            Assert.AreEqual(original.CostCenter, deserialized.CostCenter);
+            Assert.AreEqual(original.ProductGroup, deserialized.ProductGroup);
+            Assert.AreEqual(original.ProductNumber, deserialized.ProductNumber);
+            Assert.AreEqual(original.ProductBarcode, deserialized.ProductBarcode);
+            Assert.AreEqual(original.Unit, deserialized.Unit);
+            Assert.AreEqual(original.UnitQuantity, deserialized.UnitQuantity);
+            Assert.AreEqual(original.UnitPrice, deserialized.UnitPrice);
+            Assert.AreEqual(original.Currency, deserialized.Currency);
+            Assert.AreEqual(original.DecimalPrecisionMultiplier, deserialized.DecimalPrecisionMultiplier);
+        }
+
+        [Test]
+        public void BothSerializers_ProduceSameStructure()
```

Comment:
We also need a `public void BothSerializers_ProduceSameJsonString()` test for all models that checks that the two serialized json strings are equal